### ISSUE TITLE
Setting to Add Extra Arguments for Update & Install

### ISSUE THREAD

**Issue by purplespider** [alairock/Composer-AtomEditor#9]
Would it be possible to add a setting to allow users to define the additon of some extra arguments to the update/install commands?

I always use `--prefer-dist` whenever I do an install/update so it would be great to be able to specify this in settings.

The Sublime Composer module has this :)
![](http://spdr.me/1lNH8+)

**Comment by zimme** [comment]
Should be possible, but I don't have the time to implement right now.

First thought though

``` coffeescript
_run: (command, args...) ->
  ...
  args = [command, '-d', projectPath, args...]
  ...
```

This should allow the `_run` command to support arguments, then in the `install` and `update` functions you would need to check the settings to get the extra arguments.

a PR would be great.

**Comment by purplespider** [comment]
PR: #10

**Comment by zimme** [comment]
Closed with fd492556bc7380c5b9d637bf8492562096d4f9b9
